utils/hare-fetch-fids

```diff
@@ -26,11 +26,17 @@ import sys
 from typing import List, Optional, Dict, Any
 import logging
 import simplejson
+import inject
 from hax.types import ObjT
+from hax.common import di_configuration
+from hax.util import ConsulUtil, repeat_if_fails
+from hax.exception import HAConsistencyException
 
 from utils import get_node_name
 from helper.exec import Executor, Program
 
+g_kv: bool = False
```

Comment:
kindly rename appropriately, `g_kv` doesn't give much information about the flag. Some documentation would also be good.

Reply:
Done